gslb/ingestion/bootup_handler.go

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * All Rights Reserved.
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*   http://www.apache.org/licenses/LICENSE-2.0
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ingestion
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	amkov1alpha1 "github.com/vmware/global-load-balancing-services-for-kubernetes/federator/api/v1alpha1"
+	federator "github.com/vmware/global-load-balancing-services-for-kubernetes/federator/controllers"
+	"github.com/vmware/global-load-balancing-services-for-kubernetes/gslb/apiserver"
+	"github.com/vmware/global-load-balancing-services-for-kubernetes/gslb/gslbutils"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	restclient "k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var clusterScheme *k8sruntime.Scheme
+
+const (
+	metricsAddr   = ":8081"
+	apiServerPort = 9453
+)
+
+func InitializeClusterClient(cfg *restclient.Config) (client.Client, error) {
+	clusterScheme = k8sruntime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(clusterScheme))
+	utilruntime.Must(amkov1alpha1.AddToScheme(clusterScheme))
+
+	c, err := client.New(cfg, client.Options{
+		Scheme: clusterScheme,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("error in getting client: %v", err)
+	}
+
+	return c, nil
+}
+
+type AMKOClusterReconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+func createController() {
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:             clusterScheme,
+		MetricsBindAddress: metricsAddr,
+		Port:               apiServerPort,
+	})
+	if err != nil {
+		gslbutils.Errf("unable to start manager: %v", err)
```

Comment:
Can we say: `unable to start AMKOCluster controller` and similarly for all below references to `manager`? (Again, this will be changed in the federator too).

Reply:
Done